src/status_im/ui/screens/chat/views.cljs

```diff
@@ -484,23 +479,21 @@
       (let [{:keys [chat-id show-input? group-chat admins invitation-admin] :as chat}
             ;;we want to react only on these fields, do not use full chat map here
```

Comment:
I know you didn't touch this comment, but just wanted to say it's misleading. It'd be better to delete it. The destructuring of specific keys happens after the reaction is created by `re-frame.core/subscribe`, and it doesn't reduce the amount of work realized by React, Reagent or Re-frame in any way.